CMakeLists.txt

```diff
@@ -674,13 +674,15 @@ set(LIBPARQUET_SRCS
   src/parquet/arrow/record_reader.cc
   src/parquet/arrow/schema.cc
   src/parquet/arrow/writer.cc
+  src/parquet/bloom_filter.cc
   src/parquet/column_reader.cc
   src/parquet/column_scanner.cc
   src/parquet/column_writer.cc
   src/parquet/exception.cc
   src/parquet/file_reader.cc
   src/parquet/file_writer.cc
   src/parquet/metadata.cc
+  src/parquet/murmur3.cc
   src/parquet/parquet_constants.cpp
   src/parquet/parquet_types.cpp
   src/parquet/printer.cc
```

Comment:
What is this binary data file and can it be generated rather than checked into the repo?

Reply:
The binary file is used to ensure cross compatibility between parquet-mr and parquet-cpp.

Reply:
I think we should store binary files someplace else, like in a parquet-compatibility repo. I can request ASF Infra to create such a repository if you think that's a good idea

Reply:
Thanks, I agree that's a good idea. Other parquet implementation can benefit from that as well.  How long it will take to create such a repo and any dependency? I have no experience on this.